run.py

```diff
@@ -120,6 +120,8 @@ def backup_system(config, args):
     utils.copy_file(args.configfile, backup_path)
     # Backup applications
     for app in PRIMARY_APPS:
+        if app == "dovecot" and args.no_mail:
+            utils.printcolor("Skipping mail backup", utils.BLUE)
```

Comment:
A "continue" statement might be useful here?

Reply:
Hehe indeed